source/use-cases/local-master-key.txt

```diff
@@ -0,0 +1,46 @@
+A. Create a Local Master Key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB Client-Side Field Level Encryption (CSFLE) uses an encryption strategy called envelope encryption in which keys used to encrypt/decrypt data (called **data encryption keys**) are encrypted with another key (called the **master key**). For more information on the features of envelope encryption and key management concepts, see `AWS Key Management Service Concepts <https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#enveloping>`_.
+
+In CSFLE, the master key, used by the MongoDB driver to create and encrypt data keys, is securely stored in a Key Management Service (KMS). The data encryption keys, used to encrypt document fields, are stored in a user-specified collection alongside the encrypted data. 
```

Comment:
I think we can simplify how this reads by breaking it into two sentences.

In CSFLE, a MongoDB driver uses the master key to create and encrypt data data encryption keys. The master key is securely stored in a Key Management Service (KMS)...

Thoughts?

Reply:
I am fine with whatever phrasing changes. From the pattern you introduced, should it be broken into four sentences to provide symmetry?
1) what the master key does
2) where the master key is stored
3) what the data encryption keys do
4) where the data encryption keys are stored

Or maybe some other combination of three sentences?
1) what the master key does
2) where the master key is stored
3) What the data encryption keys do and where they are stored

Reply:
Resolved in person